src/tests/jsonLogic.test.js

```diff
@@ -0,0 +1,213 @@
+          rule: {
+            '>': [{ var: 'field_a' }, { '+': [{ var: 'field_b' }, { var: 'field_c' }] }],
+          },
+        },
+      });
+      const { handleValidation } = createHeadlessForm(schema, { strictInputType: false });
+      const { formErrors } = handleValidation({ field_a: 0, field_b: 1, field_c: 2 });
+      expect(formErrors.field_a).toEqual(
+        'Field A must be greater than field_b and field_b added together'
+      );
+      expect(handleValidation({ field_a: 4, field_b: 1, field_c: 2 }).formErrors).toEqual(
+        undefined
+      );
+    });
+  });
+
+  describe('Logical: ||, &&', () => {
+    it('AND: field_a > field_b && field_a > field_c (implicit with multiple rules in a single field)', () => {
+      const schema = createSchemaWithThreePropertiesWithRuleOnFieldA({
+        a_is_greater_than_b: {
+          errorMessage: 'Field A must be greater than field_b',
+          rule: {
+            '>': [{ var: 'field_a' }, { var: 'field_b' }],
+          },
+        },
+        a_is_greater_than_c: {
+          errorMessage: 'Field A must be greater than field_c',
+          rule: {
+            '>': [{ var: 'field_a' }, { var: 'field_c' }],
+          },
+        },
+      });
+      const { handleValidation } = createHeadlessForm(schema, { strictInputType: false });
+      expect(handleValidation({ field_a: 1, field_b: 10, field_c: 0 }).formErrors.field_a).toEqual(
+        'Field A must be greater than field_b'
+      );
+      expect(handleValidation({ field_a: 1, field_b: 0, field_c: 10 }).formErrors.field_a).toEqual(
+        'Field A must be greater than field_c'
+      );
+      expect(handleValidation({ field_a: 10, field_b: 5, field_c: 5 }).formErrors).toEqual(
+        undefined
+      );
+    });
+
+    it('OR: field_a > field_b or field_a > field_c', () => {
+      const schema = createSchemaWithThreePropertiesWithRuleOnFieldA({
+        field_a_is_greater_than_b_or_c: {
+          errorMessage: 'Field A must be greater than field_b or field_c',
+          rule: {
+            or: [
+              { '>': [{ var: 'field_a' }, { var: 'field_b' }] },
+              { '>': [{ var: 'field_a' }, { var: 'field_c' }] },
+            ],
+          },
+        },
+      });
+      const { handleValidation } = createHeadlessForm(schema, { strictInputType: false });
+      expect(handleValidation({ field_a: 0, field_b: 10, field_c: 10 }).formErrors.field_a).toEqual(
+        'Field A must be greater than field_b or field_c'
+      );
+      expect(handleValidation({ field_a: 1, field_b: 0, field_c: 10 }).formErrors).toEqual(
+        undefined
+      );
+      expect(handleValidation({ field_a: 10, field_b: 5, field_c: 5 }).formErrors).toEqual(
+        undefined
+      );
+    });
+  });
+
+  describe('Multiple validations', () => {
+    it('2 rules where A must be bigger than B and not an even number in another rule', () => {
```

Comment:
typo/bug: you say "and not an even number", but it's actual "and an even number" 😅 

```suggestion
    it('two rules: A > B; A is even', () => {
```